### PR TITLE
docs: fix issue #1690 instructions to change API docs

### PR DESCRIPTION
**Pull request by fozy81**
WIP

### CODE REVIEW

**Comment by fozy81** [comment]
Hi @jenweber - I've jumped in and prepared a PR, hopefully a useful starting place. Please review.

File: guides/release/contributing/index.md
Comment (ijlee2) [inline guides/release/contributing/index.md:26]: Let's use the word "update," which may sound more positive and refer to contributing to an open-source.

```suggestion
# Updating the API documentation
```

The use of `#` will create an `<h1>` tag (and `##` an `<h2>` tag, and so on). I noticed this Markdown file and 2 other files in `/release/contributing` folder misused heading levels.

In a separate pull request, would you like to help with fixing the heading levels?

File: guides/release/contributing/index.md
Comment (ijlee2) [inline guides/release/contributing/index.md:30]: Sample screenshot:



A possible update to the paragraph:

```suggestion
In the [API Guides](https://api.emberjs.com/ember/release), suppose you found a typo or wish to provide more details for a package, method, or class. Here is how you can make a change.

At the top of the page (for the package, method, or class), you will find the words "Defined in" or "Inherited from". Next to these words is a link to the source code on GitHub.

[Insert screenshot and provide alternate text if needed]

You can open the link to find a comment block. Make a pull request to update the comment block. The API Guides will be updated in the near future (e.g. on the next Ember release).
```

Comment (ijlee2) [reply guides/release/contributing/index.md:30]: @fozy81 I think adding screenshot(s) will help us concisely and precisely describe what a person will want to do to update the API Guides.

I made quite some changes to your text. Could you have a look at the suggested text, to see if it's okay with you? (Feel free to suggest something else.)

@jenweber Is it correct that, _in general_:

- Once a pull request to update the comment block is merged, the API Guides will be updated not immediately, but at the next Ember release.

File: guides/release/contributing/index.md
Comment (ijlee2) [inline guides/release/contributing/index.md:34]: Sample screenshot:

defined-in-link-for-method

A possible update to the paragraph:

```suggestion
Here is an example of updating a method. At the top of the section for [`store.createRecord()`](https://api.emberjs.com/ember-data/release/classes/Store/methods/createRecord?anchor=createRecord), you can find the words "Defined in."

[Insert screenshot and provide alternate text if needed]

Next to the words is, once again, the link to the source code [`ds-model-store.ts`](https://github.com/emberjs/data/blob/master/packages/store/addon/-private/system/ds-model-store.ts).
```

File: guides/release/contributing/index.md
Comment (ijlee2) [inline guides/release/contributing/index.md:36]: An optional suggestion. I thought the second sentence is okay, but the first sentence needed an update with regards to grammar (a run-on sentence, possibly).

```suggestion

Please follow the contributing guide in the specific repo. (Here is an example of the [contributing guide for Ember.js](https://github.com/emberjs/ember.js/blob/master/CONTRIBUTING.md).)
The contributing guide may further describe how you can edit files, commit messages, and run linting and testing.
```

**Comment by ijlee2** [comment]
@fozy81 Sorry for reviewing your pull request late and providing feedback not soon.

I thought that describing how to update the API Guides makes a great addition to the [Contributing to Ember.js](https://guides.emberjs.com/release/contributing) page. I'd like you to further iterate what you wrote, before we merge the pull request.

Can you have a look at the comments that I provided, then update the texts according to how you'd like? Thanks.

File: guides/release/contributing/index.md
Comment (jenweber) [reply guides/release/contributing/index.md:30]: I don’t think the change would necessarily be the next release since things bake in beta for 6 weeks. I’d use words like “future release” instead of “next release” to be more precise.

@ijlee2  what do you think about screenshots being optional for this to be merged? I kinda like keeping this section short and sweet for now.

Comment (ijlee2) [reply guides/release/contributing/index.md:30]: @jenweber Yep, I think we can make adding screenshots optional. 🙂

File: guides/release/contributing/index.md
Comment (fozy81) [reply guides/release/contributing/index.md]: @ijlee2 Happy to help with separate PR/issue for the headings.

**Comment by fozy81** [comment]
@ijlee2 @jenweber I've updated the wording around waiting for the future release. I've not included images for the moment - happy to add that in future if required. Thanks for the wording suggestions. Reads better now :+1: . 

Please review these updates, thanks.

**Review by ijlee2 — APPROVED** [review]
Looks great!

**Pull request merged**
